modules/busco/main.nf

```diff
@@ -0,0 +1,76 @@
+process BUSCO {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::busco=5.3.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/busco:5.3.2--pyhdfd78af_0':
+        'quay.io/biocontainers/busco:5.3.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta, stageAs: 'tmp_input/*')  // Required:    meta map, and fasta sequence files
+    each lineage                                          // Required:    lineage to check against
```

Comment:
What does `each` mean here ?

Reply:
You can give a list of lineages to test against, and it will run a task for each lineage. It's how I was running it before, but I haven't checked if you can run multiple together yet.

Reply:
e.g.
```
BUSCO ( input_ch, ['bacteria..', 'actino..' ], [], [] )
```